src/main/java/com/ode22/catnews_origins/CatGuiController.java

```diff
@@ -37,6 +47,15 @@ public class CatGuiController implements Initializable {
     @FXML
     private Label lableDate;
 
+    @FXML
+    private Label lableDate1;
```

Comment:
```suggestion
    private Label lableDate1;
```
Should be "labelDate1"